### ISSUE TITLE
the codes for the "Heat Equation"?

### ISSUE THREAD

**Issue by chengcchn** [maziarraissi/TutorialGP#1]
Hi, maziarraissi! Thanks for the nice repo.
I am interested in the example of the heat quation, but I couldn't found the responding codes. Could you please share to me?

**Comment by loskow** [comment]
Yes you are missing the k.m function that is called in a couple of places including likelihood.
